libc-top-half/musl/src/thread/pthread_mutexattr_setrobust.c

```diff
@@ -1,10 +1,13 @@
 #include "pthread_impl.h"
+#ifdef __wasilibc_unmodified_upstream
 #include "syscall.h"
```

Comment:
I wonder if we can reduce the delta from upstream by instead ifdef'ing out the entire body of `syscall.h` (it avoid ifdef'ing it out everywhere it gets included.  i.e. should we just we make including this a no-op